tests/node/index.js

```diff
@@ -0,0 +1,32 @@
+var Fs = require('fs');
+var Path = require('path');
+var Env = require('../..').Env;
+var Fetch = require('../..').fetch;
+
+
+var env = new Env('en-US', Fetch);
+var ctx = env.createContext([
+  Path.join(__dirname + '/fixtures/{locale}/entities.l20n')
+]);
+var langs = [
+  {code: 'es-ES'},
+  {code: 'en-US'},
+  {code: 'fr'}
+];
+
+ctx.fetch(langs).then(init);
+
+function init () {
+  ctx.resolve(langs, 'whatCanIAsk').then(function (value) {
+    console.log('SINGLE VALUE: ', value);
+  });
+
+  Promise.all([
+    ctx.resolve(langs, 'whatCanIAsk'),
+    ctx.resolve(langs, 'openApp'),
+    ctx.resolve(langs, 'callNumber'),
+    ctx.resolve(langs, 'notFound')
+  ]).then(function (values) {
+    console.log('MULI VALUES', values);
+  });
```

Comment:
The `langs` comment I made above seems more obvious when we want multiple entity values.

I spoke with @zbraniecki a couple days ago about being able to pass an array of entities instead of using `Promise.all`. He liked the idea but also mentioned others may not.

Another thing I don't like much about using `Promise.all` is that the results come back in an array. Also, when getting multiple entities resolved, I'd prefer to have the results as a dictionary instead, since I'd probably end up producing one myself for easier digestion in my app.

Maybe multiple entity results could look something like this.

```js
{
  whatCanIAsk: { value: '¿Qué puedo pedir?', attrs: null },
  openApp: { value: 'Open &lt;App&gt;', attrs: null },
  callNumber: { value: '&lt;Número&gt; De Llamada', attrs: null },
  notFound { value: 'notFound', attrs: null }
}
```

To speak further to translating multiple entities... I could imagine producing a huge array of `ctx.resolve(langs, 'someKey')` and producing a smell in my code. I'm thinking of an interface where all my entities were resolved at once, without defining each of them in my call to `resolve` (they are already defined in my `.l20n` files)... and getting a dictionary back like shown above with all my entities resolved.